index.src.html

```diff
@@ -2914,11 +2949,14 @@
             {{Plane/stride}}, {{Plane/rows}}, {{Plane/length}}.
         4. Append |plane| to |planes|.
     6. Assign |planes| to |frame|.{{VideoFrame/planes}}.
-    7. If {{VideoFrameInit/duration}} [=map/exists=] in |init|, assign it to
+    7. If |otherFrame|.{{VideoFrame/hasAlpha}} is `true` and
+        |init|.{{VideoFrameInit/alpha}} is `"discard"`, run the
+        [=Discard alpha=] algorithm with |frame|.
```

Comment:
I think we might drop the discard alpha algo entirely. as-is, it seems to mutate the underlying resource for both the original and new frame. But also, I think given how the dust is settling for interleaved, per dfn of hasAlpha above, we can allow the two frames to simply share the same resource and rely on the definition of hasAlpha + format to do the right thing. For this method, you could then just say something like:

if alpha = discard, assign `false` to hasAlpha. otherwise, assign otherframe.hasAlpha to hasAlpha.